test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -2283,7 +2283,7 @@ public void testMetadataColumns_timestampNull() throws Exception {
 
   @Test
   public void testMetadataColumns_clob() throws Exception {
-    assumeFalse("MySQL does not support clobs", is(MYSQL));
+    // CLOB is TEXT in MySQL
```

Comment:
I don't really get the point of these comments here. It's not specific to this test, and is part of the swizzling in JdbcFixture. I also wasn't sure about the NCLOB reference, since there is no NCLOB anywhere in the tests.